editors/vscode/syntaxes/lp.tmLanguage.json

```diff
@@ -44,7 +44,7 @@
     },
 
     "signature-commands": {
-      "match": "(definition|as|in|let|declared|builtin|proof|qed|rule|symbol|theorem|with)\\s+([^\\s+]*)",
+      "match": "(begin|end|definition|as|in|let|declared|builtin|rule|symbol|with)\\s+([^\\s+]*)",
```

Comment:
The keyword "begin" has the first 3 letters in one color and the last 2 letters in another color. Perhaps you need to put the keyword "begin" after the keyword "in". If this is the case, thanks to add a comment saying that the order is important and mentioning this example.

Reply:
This is weird, that didn't happen in my vscode

Reply:
Could you get the version 1.52.0?

Reply:
Didn't see that, I should refresh before asking.

I still couldn't get the same error. 
Its weird, as both `begin` and `in` are in the same `signature-commands` so  they should have the same colour

Reply:
Sorry, this is my mistake: I was not testing the right branch.